package.json

```diff
@@ -765,14 +786,19 @@
                     "when": "view =~ /(azureResourceGroups|azureFocusView)/ && viewItem =~ /userAssignedIdentity/",
                     "group": "1@1"
                 },
+                {
+                    "command": "azureFunctions.durableTaskScheduler.copyEmulatorConnectionString",
+                    "when": "view =~ /(azureWorkspace|azureFocusView)/ && viewItem =~ /azFunc.dts.emulatorInstance/",
```

Comment:
This probably doesn't need `azureFocusView` in the view regex.

Reply:
Updated.